src/components/global/Widget/index.tsx

```
@@ -0,0 +1,59 @@
+        try {
+          const profile = await fetchUserProfile(user.id);
+          setProfile(profile);
+        } catch (error) {
+          console.error('Error fetching user profile:', error);
+        }
+      })();
+    }
+  }, [user]);
+
+  useEffect(() => {
+    if (user && user.id) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)